src/core/blockchain/types/client.ts

```diff
@@ -36,6 +37,7 @@ export abstract class BlockchainGenericClient {
     public abstract calculateFees(
         from: string,
         to: string,
+        tokenType: TokenType,
```

Comment:
move this at the end, and leave it optional, this way we could use this method to estimate costs on all our smart contract calls not only token transfers.

also give a default value:
```
tokenType: TokenType = TokenType.NATIVE
```